aioxmpp/muc/service.py

```diff
@@ -618,18 +633,28 @@ def _match_tracker(self, message):
         try:
             tracker = self._tracking_by_id[message.id_]
         except KeyError:
+            key = message.from_, _extract_one_pair(message.body)
```

Comment:
Ok, I restate this, nick changes are problematic with tracking in all cases. You convinced me dropping `from_` is the way to go, it seems to be the most robust heuristic.